contracts/vaults/Base.sol

```diff
@@ -274,7 +279,7 @@ contract Base is OwnableUpgradeable, ReentrancyGuardUpgradeable {
     }
 
     modifier noSubmittedClaims(uint256 _pid) {
```

Comment:
maybe also rename the modifier at this point to "noActiveClaims"

Reply:
i was thinking to change the struct and mapping too